pinot-integration-tests/src/test/resources/On_Time_On_Time_Performance_2014_100k_subset.test_queries_200.sql

```diff
@@ -117,54 +117,54 @@
+{"sql":"SELECT SUM(DivArrDelay), SUM(DestStateFips) FROM mytable WHERE DivTailNums IN ('N3768', 'N438SW', 'N786SK', 'N636MQ', 'N3KXAA') AND DestAirportID < 11867 OR Carrier NOT IN ('DL', 'WN') LIMIT 20","hsqls":["SELECT SUM(DivArrDelay), SUM(DestStateFips)  FROM mytable WHERE (DivTailNums[1] IN ('N3768', 'N438SW', 'N786SK', 'N636MQ', 'N3KXAA') OR DivTailNums[2] IN ('N3768', 'N438SW', 'N786SK', 'N636MQ', 'N3KXAA') OR DivTailNums[3] IN ('N3768', 'N438SW', 'N786SK', 'N636MQ', 'N3KXAA') OR DivTailNums[4] IN ('N3768', 'N438SW', 'N786SK', 'N636MQ', 'N3KXAA') OR DivTailNums[5] IN ('N3768', 'N438SW', 'N786SK', 'N636MQ', 'N3KXAA')) AND DestAirportID < 11867 OR Carrier NOT IN ('DL', 'WN') LIMIT 10000"]}
+{"sql":"SELECT SUM(DivReachedDest), AVG(DestWac) FROM mytable WHERE DivWheelsOns BETWEEN 728 AND 1804 LIMIT 13","hsqls":["SELECT SUM(DivReachedDest), AVG(DestWac)  FROM mytable WHERE (DivWheelsOns[1] BETWEEN 728 AND 1804 OR DivWheelsOns[2] BETWEEN 728 AND 1804 OR DivWheelsOns[3] BETWEEN 728 AND 1804 OR DivWheelsOns[4] BETWEEN 728 AND 1804 OR DivWheelsOns[5] BETWEEN 728 AND 1804) LIMIT 10000"]}
+{"sql":"SELECT SUM(WheelsOn), SUM(DivArrDelay), COUNT(DestState) FROM mytable WHERE DivAirports IN ('RFD') AND TaxiIn BETWEEN -9999 AND 94 LIMIT 29","hsqls":["SELECT SUM(WheelsOn), SUM(DivArrDelay), COUNT(DestState)  FROM mytable WHERE (DivAirports[1] IN ('RFD') OR DivAirports[2] IN ('RFD') OR DivAirports[3] IN ('RFD') OR DivAirports[4] IN ('RFD') OR DivAirports[5] IN ('RFD')) AND TaxiIn BETWEEN -9999 AND 94 LIMIT 10000"]}
 {"sql":"SELECT SUM(\"Month\") FROM mytable WHERE FirstDepTime <> 1742 AND DestStateName BETWEEN 'Maine' AND 'South Dakota' OR Carrier <= 'UA' LIMIT 23","hsqls":["SELECT SUM(`Month`) FROM mytable WHERE FirstDepTime <> 1742 AND DestStateName BETWEEN 'Maine' AND 'South Dakota' OR Carrier <= 'UA' LIMIT 23"]}
-{"sql":"SELECT SUM(\"Year\"), MAX(DayofMonth), AVG(CRSArrTime) FROM mytable WHERE WheelsOff > 1737 AND DivTotalGTimes IN (30, 113, 102, 37) LIMIT 10","hsqls":["SELECT SUM(`Year`), MAX(DayofMonth), AVG(CRSArrTime)  FROM mytable WHERE WheelsOff > 1737 AND (DivTotalGTimes__MV0 IN (30, 113, 102, 37) OR DivTotalGTimes__MV1 IN (30, 113, 102, 37) OR DivTotalGTimes__MV2 IN (30, 113, 102, 37) OR DivTotalGTimes__MV3 IN (30, 113, 102, 37) OR DivTotalGTimes__MV4 IN (30, 113, 102, 37)) LIMIT 10000"]}
+{"sql":"SELECT SUM(\"Year\"), MAX(DayofMonth), AVG(CRSArrTime) FROM mytable WHERE WheelsOff > 1737 AND DivTotalGTimes IN (30, 113, 102, 37) LIMIT 10","hsqls":["SELECT SUM(`Year`), MAX(DayofMonth), AVG(CRSArrTime)  FROM mytable WHERE WheelsOff > 1737 AND (DivTotalGTimes[1] IN (30, 113, 102, 37) OR DivTotalGTimes[2] IN (30, 113, 102, 37) OR DivTotalGTimes[3] IN (30, 113, 102, 37) OR DivTotalGTimes[4] IN (30, 113, 102, 37) OR DivTotalGTimes[5] IN (30, 113, 102, 37)) LIMIT 10000"]}
```

Comment:
Can we make this query to be something like for h2:
```
...
WHERE EXISTS (
    SELECT 1
    FROM unnest(DivTotalGTimes) AS DivTotalGTime
    WHERE DivTotalGTime = ANY(ARRAY[30, 113, 102, 37])
);
```

Reply:
I gave it a try but couldn't able to come up with a query. If you can, please share the working query; I will update the PR. Thanks

Reply:
I also tried and it somehow gives `Column "DIVTOTALGTIMES" not found` syntax error. cc @xiangfu0

Reply:
Yes, it requires `Select` inside the `unnest` function, which again requires a `Where` clause to get the unique row. 
In my opinion, If we get the working query, It will be more complex.

Reply:
I dig a bit one thing is to change the query to use `ARRAY_CONTAINS`, e.g.
```
{"sql":"SELECT SUM(\"Year\"), MAX(DayofMonth), AVG(CRSArrTime) FROM mytable WHERE WheelsOff > 1737 AND DivTotalGTimes IN (30, 113, 102, 37) LIMIT 10","hsqls":["SELECT SUM(`Year`), MAX(DayofMonth), AVG(CRSArrTime)  FROM mytable WHERE (WheelsOff > 1737) AND (ARRAY_CONTAINS(DivTotalGTimes, 30) OR ARRAY_CONTAINS(DivTotalGTimes, 37) OR ARRAY_CONTAINS(DivTotalGTimes, 102) OR ARRAY_CONTAINS(DivTotalGTimes, 113)) LIMIT 10000"]}
```

Reply:
I haven't find a way to use one stmt to merge those OR clause.

Reply:
True, the `ARRAY_CONTAINS` function only searches for one value. Let me change to the `ARRAY_CONTAINS` style.

Reply:
Thanks! I think it's better than using array index.